helm/auto-deployment/bold-bi-auto-deployment-using-helm.md

```diff
@@ -0,0 +1,439 @@
+# Bold BI Auto Deployment using Helm
+This section helps you to deploy [Bold BI](https://www.boldbi.com/) using helm chart without manually activating license and configuring the startup from browser. You can pass the license, startup and branding related customization as environment variable in values.yaml file.
+
+## Deployment prerequisites
+
+* [Install Helm](https://helm.sh/docs/intro/install/) to deploy Bold BI using Helm.
+* [File Storage](../docs/pre-requisites.md#file-storage)
+* [Create and connect a cluster](../docs/pre-requisites.md#create-a-cluster)
+* [Load Balancing](../docs/pre-requisites.md#load-balancing)
+
+**Note:** Note the [Ingress IP address](docs/pre-requisites.md#get-ingress-ip) and map it with a DNS to crafting values.yaml when installing Bold BI with helm chart.
```

Comment:
Note the [Ingress IP address](docs/pre-requisites.md#get-ingress-ip) and map it with a DNS to craft values.yaml when installing Bold BI with helm chart.

Reply:
concern addressed